### PR TITLE
fix(1858): the path of meta.summary is wrong

### PR DESCRIPTION
**Pull request by wahapo**
## Context
I made a mistake in my previous PR https://github.com/screwdriver-cd/launcher/pull/384. The correct one is `meta.summary`.

## References
https://github.com/screwdriver-cd/screwdriver/issues/1858

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: launch.go [hunk 1]
```diff
@@ -494,7 +494,9 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 	}
 
 	// Initialize pr comments (Issue #1858)
-	delete(mergedMeta, "summary")
+	if _, ok := mergedMeta["meta"]; ok {
+		delete(mergedMeta["meta"], "summary")
```
Comment (wahapo) [inline launch.go:498]: I can't get it to work here. Does anyone know what the solution is?
```
./launch.go:498:9: first argument to delete must be map; have interface {}
```

Comment (tkyi) [inline launch.go:498]: Probably need to cast it like this
```suggestion
	if metadata, ok := mergedMeta["meta"]; ok {
		delete(metadata.(map[string]interface{}), "summary")
```
https://github.com/screwdriver-cd/launcher/pull/383/files#diff-0952bc032b3245f7e7282a5d24e8a1775eb3e1f265de43bb26834860e6acc4edR318

Comment (wahapo) [reply launch.go]: Thanks!

**Review by yoshwata — APPROVED** [review]

**Review by tkyi — APPROVED** [review]

**Comment by wahapo** [comment]
@tkyi Thanks for the PR merge😃 
A new [launcher@v6.0.105](https://github.com/screwdriver-cd/launcher/releases/tag/v6.0.105) has been generated, and I want to reflect it in <https://cd.screwdriver.cd/>. What do I need to do to reflect this? Would you please reflect this?

**Pull request merged**